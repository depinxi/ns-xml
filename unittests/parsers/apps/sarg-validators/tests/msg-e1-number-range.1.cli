msg "Testing number range validation (min)" --range 1