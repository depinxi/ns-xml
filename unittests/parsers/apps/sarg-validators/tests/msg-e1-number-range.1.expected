CLI: "msg", "Testing number range validation (min)", "--range", "1"
Value count: 1
Values: "Testing number range validation (min)"
Error count: 1
Subcommand: msg
argumentNumber=
argumentNumberPositive=
argumentNumberNegative=
argumentNumberRange=
argumentNumberNegativeRange=
argumentSelect=
argumentNumberSelect=
